### PR TITLE
Upload Download Pipeline Example

### PR DESCRIPTION
**Pull request by diegolovison**
Automated test for: https://github.com/opendatahub-io/data-science-pipelines-operator/pull/228

```
Verify Ods Users Can Create And Run A Data Science Pipeline Using ... | PASS |
------------------------------------------------------------------------------
Tests.Ods Dashboard.Data Science Pipelines.Data-Science-Pipelines-... | PASS |
1 test, 1 passed, 0 failed
==============================================================================
Tests.Ods Dashboard.Data Science Pipelines                            | PASS |
1 test, 1 passed, 0 failed
==============================================================================
Tests.Ods Dashboard                                                   | PASS |
1 test, 1 passed, 0 failed
==============================================================================
Tests                                                                 | PASS |
1 test, 1 passed, 0 failed
==============================================================================
Output:  /home/dlovison/github/openshift-data-science/ods-ci/ods_ci/test-output/ods-ci-2023-08-16-10-15-pXoGtsYNcQ/output.xml
XUnit:   /home/dlovison/github/openshift-data-science/ods-ci/ods_ci/test-output/ods-ci-2023-08-16-10-15-pXoGtsYNcQ/xunit_test_result.xml
Log:     /home/dlovison/github/openshift-data-science/ods-ci/ods_ci/test-output/ods-ci-2023-08-16-10-15-pXoGtsYNcQ/log.html
Report:  /home/dlovison/github/openshift-data-science/ods-ci/ods_ci/test-output/ods-ci-2023-08-16-10-15-pXoGtsYNcQ/test_report.html

```

### CODE REVIEW

**Review by lugi0 — APPROVED** [review]

**Review by jgarciao — APPROVED** [review]

**Comment by bdattoma** [comment]
@diegolovison could you plz check the security scan result?

**Comment by diegolovison** [comment]
> @diegolovison could you plz check the security scan result?

>Make sure publicly writable directories are used safely here.

writing to tmp looks fine to me

File: poetry.lock
[the remark this answers is not in the document]
Comment (diegolovison) [reply poetry.lock:1]: What do you think https://jenkins/job/rhods-smoke/3117/TestReport/

[the remark this answers is not in the document]
Comment (jgarciao) [reply poetry.lock:1]: 2 tests failed but I think it was unrelated to this changes. Ok for me

**Pull request merged**